www/docs/en/11.x/core/features/splashscreen/index.md

```diff
@@ -351,12 +391,16 @@ Amount of time in milliseconds to wait before automatically hide splash screen.
 
 **Supported Platforms:**
 
+- Android
 - iOS
 
 **Data Type:** `Number`, in milliseconds
 
 **Default Value:** `true`
 
+- Android
+  `-1`: The splash screen will automatticly hide when the `onPageFinished` has been triggered.
+
 - iOS
   `3000`: The splash screen will automatticly hide in 3 seconds.
```

Comment:
```suggestion
  `3000`: The splash screen will automatically hide in 3 seconds.
```

I know this wasn't changed in this PR, but has the same spelling mistake, so may as well correct it at the same time.